corehq/apps/app_manager/suite_xml.py

```diff
@@ -1913,6 +1914,44 @@ def get_target_module(case_type, module_id, with_product_details=False):
             except IndexError:
                 pass
 
+        self.add_parent_datums(datums, module)
+
+        return datums, assertions
+
+    def add_parent_datums(self, datums, module):
+
+        def update_refs(action_, datum_, changed_ids):
+            if action:
```

Comment:
This is not referencing a local variable (although `action_` looks very similar). Is that intentional?